frameworks/CSharp/aspnetcore/benchmark_config.json

```diff
@@ -75,6 +75,25 @@
       "notes": "",
       "versus": "aspcore"
     },
+    "mw-utf8json": {
+      "plaintext_url": "/plaintext",
```

Comment:
Shouldn't have `plaintext_url` only `json_url` would be no variation between `aspcore-mw` and `aspcore-mw-utf8json`; so add no value (as its an alternative json rather than plaintext test being added).

Also will probably fail as the Plaintext scenario is not added in the docker